lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -189,8 +195,10 @@ def configure(conf)
         # Check for specialized GCE environments (Managed VM or Dataflow).
         # TODO: Add config options for these to allow for running outside GCE?
         attributes = fetch_gce_metadata('instance/attributes/').split
-        if attributes.include?('gae_backend_name') &&
-           attributes.include?('gae_backend_version')
+        if @force_compute_service
```

Comment:
Yeah, I'm not the biggest fan either, but it seemed the best framing of it I could come up with. It's better than a string field that lets you pick the service name because we may not be able to populate the other fields for that service name (or even know what it is). It's more general than a parameter specifically for disabling the container service and could potentially be useful in other similar cases. Also, naming a parameter that would do this but only for the container service would seem to be even harder. ignore_kube_env could perhaps work, but kube-env is more of an implementation detail than something most people know anything about.

Do you have any ideas?